src/CacheKey.php

```diff
@@ -97,7 +97,7 @@ protected function getQueryColumns(array $columns) : string
 
     protected function getTypeClause($where) : string
     {
-        $type = in_array($where["type"], ["In", "NotIn", "Null", "NotNull", "between", "NotInSub", "InSub", "JsonContains"])
+        $type = in_array($where["type"], ["InRaw", "In", "NotIn", "Null", "NotNull", "between", "NotInSub", "InSub", "JsonContains"])
```

Comment:
I think the other new `types` should be supported here too:
`"InRaw", "NotInRaw", "IntegerInRaw", "IntegerNotInRaw"`

And for completeness probably should be added to the array in `getOtherClauses` too.

Reply:
Will also need to get the tests written.